service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -302,24 +302,24 @@ public final DeviceRegistrationClient getRegistrationClient() {
     }
 
     /**
-     * Sets the factory to use for creating a client for the Credentials service.
+     * Sets the client to use for accessing the Credentials service.
      *
-     * @param factory The factory.
+     * @param client The client.
      * @throws NullPointerException if the factory is {@code null}.
```

Comment:
In  `@throws NullPointerException if the factory is {@code null}.` the _factory_ should be replaced with _client_.